hyperstart_agent.go

```diff
@@ -314,7 +315,22 @@ func (h *hyper) exec(pod *Pod, c Container, cmd Cmd) (*Process, error) {
 		Process:   *hyperProcess,
 	}
 
-	process, err := prepareAndStartShim(pod, h.shim, c.id, token, h.state.URL, cmd)
```

Comment:
I prefer passing additional arguments here rather than duplicating the code.

Reply:
Fair enough ! I have removed it.